cms/static/js/models/settings/course_details.js

```diff
@@ -28,14 +28,21 @@ var CourseDetails = Backbone.Model.extend({
         // Returns either nothing (no return call) so that validate works or an object of {field: errorstring} pairs
         // A bit funny in that the video key validation is asynchronous; so, it won't stop the validation.
         var errors = {};
+        newattrs = DateUtils.convertDateStringsToObjects(
+            newattrs, ["start_date", "end_date", "enrollment_start", "enrollment_end"]
+        );
+
         if (newattrs.start_date === null) {
             errors.start_date = gettext("The course must have an assigned start date.");
         }
+        if (this.hasChanged("start_date") && this.get("has_cert_config") === false){
+            errors.start_date = gettext("The course must have at least one active certificate configuration before it can be started.");
+        }
```

Comment:
Value of `has_cert_config` would be `null` if cert feature is either turned off for system or for course. Its value would if `false` if feature is turned on both system wide and for course but there is no certificate configuration available and if there is a configuration available its value would be `true`.